### ISSUE TITLE
did we support arweave new version 2.6？

### ISSUE THREAD

**Issue by rosebush** [virdpool/miner#27]
in 3.6， will hard fork

**Comment by vird** [comment]
No. 2.6 is very different. Pool should give you address, you specify that address for repack and only then you can mine.
It requires full rework of pool, miner
